packages/livekit-rtc/src/audio_frame.ts

```diff
@@ -11,6 +11,11 @@ export class AudioFrame {
   channels: number;
   samplesPerChannel: number;
 
+  // note: if converting from Uint8Array to Int16Array, *do not* use buffer.slice!
+  // it is marked unstable by Node and can cause undefined behaviour, such as massive chunks of
+  // noise being added to the end.
+  // it is recommended to use buffer.subarray instead.
+  // XXX(nbsp): add this when writing proper docs
```

Comment:
maybe also add to the readme code example? https://github.com/livekit/node-sdks/tree/main/packages/livekit-rtc